src/components/DropDownList/DropDownList.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Person } from '../../types/Person';
+import { Gender } from '../../types/Gender';
+
+type Props = {
+  suggestedPeople: Person[];
+  onSelected: (person: Person) => void;
+};
+
+export const DropDownList: React.FC<Props> = React.memo(
+  ({ suggestedPeople, onSelected }) => {
+    return (
+      <div className="dropdown-menu" role="menu">
+        <div className="dropdown-content">
+          {suggestedPeople.length
+            ? (
+              suggestedPeople.map(suggestedPerson => (
+                <a
+                  key={suggestedPerson.name}
```

Comment:
Using the person's name as a key may lead to problems if there are people with the same name. Consider using a unique identifier, like an ID, for the key prop.